test/integration/user-management-api/applications_test.rb

```diff
@@ -80,220 +83,188 @@ def setup
   test 'index is paginated' do
     # creating more apps
     #TODO move app creation to helpers...
-    buyer = FactoryBot.create(:buyer_account, :provider_account => @provider)
+    buyer = FactoryBot.create(:buyer_account, provider_account: @provider)
     buyer.buy! @provider.default_account_plan
-    @application_plan = FactoryBot.create :application_plan, :issuer => @provider.first_service!
+    @application_plan = FactoryBot.create :application_plan, issuer: @provider.first_service!
     @application_plan.publish!
     buyer.buy! @application_plan
 
-    get(admin_api_applications_path(:format => :xml), :provider_key => @provider.api_key,
-             :per_page => 1)
+    get admin_api_applications_path(format: :xml), params: { provider_key: @provider.api_key, per_page: 1 }
 
     assert_response :success
     assert_pagination @response.body, "applications"
   end
 
   test 'pagination per_page has a maximum allowed' do
-    # Two applications because pagination should be shown only if needed, so total_entries > per_page
-    application_plan = FactoryBot.create :application_plan, :issuer => @provider.first_service!
+    # Two applications because pagination test be shown only if needed, so total_entries > per_page
+    application_plan = FactoryBot.create :application_plan, issuer: @provider.first_service!
     @buyer.buy! application_plan
 
-    max_per_page = set_api_pagination_max_per_page(:to => 1)
+    max_per_page = set_api_pagination_max_per_page(to: 1)
 
-    get(admin_api_applications_path(:format => :xml), :provider_key => @provider.api_key,
-             :per_page => (max_per_page +1))
+    get admin_api_applications_path(format: :xml), params: { provider_key: @provider.api_key, per_page: (max_per_page +1) }
 
     assert_response :success
-    assert_pagination @response.body, "applications", :per_page => max_per_page
+    assert_pagination @response.body, "applications", per_page: max_per_page
   end
 
   test 'pagination page defaults to 1 for invalid values' do
-    # Two applications because pagination should be shown only if needed, so total_entries > per_page
-    application_plan = FactoryBot.create :application_plan, :issuer => @provider.first_service!
+    # Two applications because pagination test be shown only if needed, so total_entries > per_page
```

Comment:
original wording sounds clearer to me, just saying, not a blocker

Reply:
Oops, I did a global replace and overlooked this one.